cpu/lpc2387/include/cpu_conf.h

```diff
@@ -42,11 +42,10 @@ extern "C" {
  * @name Kernel configuration
  * @{
  */
-#define THREAD_EXTRA_STACKSIZE_PRINTF_FLOAT  (4096)
-#define THREAD_EXTRA_STACKSIZE_PRINTF        (2048)
+#define THREAD_EXTRA_STACKSIZE_PRINTF        (512)
 
 #ifndef THREAD_STACKSIZE_DEFAULT
-#define THREAD_STACKSIZE_DEFAULT   (512)
+#define THREAD_STACKSIZE_DEFAULT   (1024)
 #endif
 
 #define THREAD_STACKSIZE_IDLE      (160)
```

Comment:
Should this also be updated to the Cortex M value (256) for consistency? It apparently worked without. It would be nice to finally get rid of the idle thread at all to safe some RAM and just add `while(1) {pm_lowset()}` directly into the scheduler when no thread is runnable. But that is an unrelated discussion.

Reply:
So far 160 bytes have been enough an `ps` shows 152 are used.